microreactors/gcmr/core/Neutronics/tests

```diff
@@ -0,0 +1,30 @@
+[Tests]
```

Comment:
you ll need a first 'RunApp' test that generates the mesh

then get rid of parallel scheduling because that first test must run first

Reply:
Added a RunApp to generate the mesh,